### DISCUSSION TITLE
Can I add a study selected by myself after I entered oracle mode

### DISCUSSION THREAD

**Discussion by ReintHuis** [asreview/asreview#1266] [Q&A] [answered]
Can I add  a study (within or outside of the dataset) that I select myself, after the phase of adding prior knowledge? Currently I am working on a search term in ASreview, but I have found a new relevant study trough Google Scholar that I wish to add to ASreview, because I believe it will help to train the algorithm.  Thanks in advance!

**Comment by Rensvandeschoot** [comment, marked as answer]
Currently, there is no feature to do this in the software, but what you could do is easy:

1) Export your data (the labels for the priors will be [exported](https://asreview.readthedocs.io/en/latest/data_labeled.html#label-format))
2) Add records you want to add in reference manager software (Keep an eye on the `datatool` repository for functionality to compare and stack multiple datasets: https://github.com/asreview/asreview-datatools/pulls)
3) Start a new project, add the data, and the previously labeled studies will be recognized as prior knowledge
